grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java

```diff
@@ -430,7 +431,7 @@ private Json serialiseStateFull(TaskState state) {
                 .set("interval", state.schedule().interval().map(Duration::toMillis).orElse(null))
                 .set("recurring", state.schedule().isRecurring())
                 .set("exception", state.exception())
```

Comment:
Oooo here they are hiding. As hard coded strings???? FLOGGINGS!!!!!! _starts foaming at the mouth_